### DISCUSSION TITLE
Change in narration-style

### DISCUSSION THREAD

**Discussion by Dinock** [GDQuest/learn-gdscript#134] [Q&A]
I wasn't sure if this is a bug or intended so I opened a discussion instead of an issue.

In the last paragraphs of lesson 7 the narrative switches from an "I" to a "we". Before this point I always imagined a fictional person teaching and telling me stuff about coding. But at the end of lesson 7 the narration is changed to as if there were more than one person teaching me how to code.

I am not sure if this was made on purpose or by mistake. Nonetheless did it irritate me a bit while reading.

Screenshot of the paragraphs in question (lesson 7):
![typo-we_anstatt_I](https://user-images.githubusercontent.com/54438803/148703661-2f4d2364-0608-45de-80de-50040bccf3d7.png)

---

So after writing the text above I reread some of the lessons again and now I am totally unsure if the narration-style is consistent or not.

Screenshot of an "I"-narration (lesson 8):
![lesson8_I](https://user-images.githubusercontent.com/54438803/148704740-bab868a8-698f-4716-9c0e-f4da80a7fdf6.png)

To me it seems to be something like this: The narrator is Nathan who is speaking for the whole GDQuest-Team, similar to what it is like in a YouTube-Tutorial made by the entire GDQuest-Team but narrated by Nathan.

So maybe I am just overthinking this.
But I am also interested in other opinions/how other people who finished the beta-version of this course experienced this.

**Comment by Mark-Karnowski** [comment]
Why use first person pronouns at all?

"A future lesson will explain variables and subvariables and their relationship to one another."

"For now, practice accessing variables."

"As hinted above, in GDScript variables work a bit like labels."


---------------------------------------------------------------------

The course is still in the very early stages of development. I'm sure that sure that edting and revising the texts will be part of the process at some point.

**Reply by NathanLovato** [reply]
The first person makes it more personal and a bit less cold.

**Reply by Mark-Karnowski** [reply]
I don't entirely disagree, but I've been examining the writng style in a series of introductory coding books "[in easy steps](https://ineasysteps.com/)". In the books I looked through, the authors never use "I".  They use "we" very infrequently, and when they do it is **always** in the inclusive sense, that is, including the reader, as in ..."we've created a spiral, and now we need to add a background."  On the other hand, they use the second person very liberally, as in "..upload a picture you have already made" or "in the last lesson you learned how to move the sprite." It seems to me that the use of the inclusive "we" and frequent use of the pronoun "you"  is what gives them their personal tone and keeps the focus on the learner, rather than on the teacher.

**Reply by NathanLovato** [reply]
We avoid the passive voice to make the writing more accessible, especially to non-native speakers, which is why, unlike in the book you linked, we have more pronouns: with the direct voice, you need a subject.

I don't mind removing the subject when it's not necessary, of course. It's a small detail.

Then, changing I into we in other cases... I don't really care. The way we wrote the lessons, that'll sound like there are multiple tutors rather than one. It won't make it sound like we're including the student: we use the second person for what the student does.

To me, the task right now is to:

- Remove "I" when it's not necessary.
- Settle on using I or we when talking from the tutors' point of view. We can use whichever, it'll make little difference the way we write the lessons.

Ping @johnnygossdev so you can join the discussion.

**Reply by cogwoz** [reply]
I think it should be consistant of course. 

I prefer using "we" for the tutors' point of view for the reasons mentioned by @Mark-Karnowski.

I agree with moving forward with the tasks you mentioned. I can do a sweep along with the next round of lessons.

**Comment by NathanLovato** [comment]
Sure, it's an inconsistency. I'll open an issue to address this.
